lib/srv/db/spanner_test.go

```diff
@@ -241,45 +242,39 @@ func TestAuditSpanner(t *testing.T) {
 			reconnectingCh <- clt.ClientConn.WaitForStateChange(ctx, connectivity.Ready)
 		}()
 
-		ev := requireEvent(t, testCtx, libevents.DatabaseSessionStartFailureCode)
-		dbStart1, ok := ev.(*events.DatabaseSessionStart)
-		require.True(t, ok)
-		require.Equal(t, "googlesql", dbStart1.DatabaseName)
-
-		row, err := pingSpanner(ctx, clt, 42)
-		require.Error(t, err)
-		require.ErrorContains(t, err, "access to db denied")
-		require.Nil(t, row)
-
-		ev = requireEvent(t, testCtx, libevents.DatabaseSessionStartFailureCode)
-		dbStart2, ok := ev.(*events.DatabaseSessionStart)
-		require.True(t, ok)
-		require.Equal(t, "googlesql", dbStart1.DatabaseName)
-
 		require.True(t, <-reconnectingCh, "timed out waiting for the spanner client to reconnect")
```

Comment:
is `<-reconnectingCh` still needed? like maybe just run `clt.ClientConn.WaitForStateChange` without the goroutine.